wrangler-service/src/main/java/co/cask/wrangler/service/explorer/FilesystemExplorer.java

```diff
@@ -115,6 +151,12 @@ public void read(HttpServiceRequest request, HttpServiceResponder responder,
                    @QueryParam("sampler") String sampler,
                    @QueryParam("fraction") double fraction,
                    @QueryParam("scope") String scope) {
+      
+    LOG.debug("Read API. HTTP Request headers: {}", 
+            Arrays.toString(request.getAllHeaders().entrySet().toArray()));
+    LOG.debug("Read API. Query Params: Path {}, lines {}, sampler {}, fraction {}, scope {}",
+            path, lines, sampler, fraction, scope);
+
```

Comment:
do we need them, consider removing them.

Reply:
Keeping it for now